docs/workflows/manage-content.rst

```diff
@@ -73,6 +74,27 @@ with a duplicate name will first remove the existing tag
 
 Reference: `Docker Recursive Add Usage <../restapi.html#tag/docker:-recursive-add>`_
 
+.. _recursive-remove:
+
+Recursively Remove Content from a Repository
+--------------------------------------------
+
+Any Docker content can be removed from a repository version with the
+recursive-remove endpoint. Recursive remove is symmetrical with
+recursive add, meaning that performing a recursive-add and a
+recursive-remove with the same content will result in the original
+content set.
```

Comment:
Nitpick: There are subtle cases where this is not the case (w.r.t. "idempotence" of operations):

1. Assume that the latest repo versions contains: Manifest M -> Blob B
2. User adds Tag T recursively, T points to M. Repo version contains T -> M -> B
3. User removes T recursively. Repo version is empty.